README.md

```diff
@@ -39,7 +39,7 @@ v7.10.0
 $npm -v
 4.2.0
 
-$sudo npm install -g @angular/cli@1.3.1		
+$npm install -g @angular/cli@1.3.1		
```

Comment:
Does the non sudo version of this setup the PATH properly for the "ng" commands?

Reply:
ng commands (or any commands installed globally using npm) should work in any directory assuming that npm is set up correctly.

Reply:
Assuming using the specific apt-get install command above?